app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/connectors/TestCreateJourneyConnector.scala

```diff
@@ -70,7 +70,8 @@ class TestCreateJourneyConnector @Inject()(httpClient: HttpClient,
 object TestCreateJourneyConnector {
   implicit val journeyConfigWriter: Writes[JourneyConfig] = (journeyConfig: JourneyConfig) => Json.obj(
     continueUrlKey -> journeyConfig.continueUrl,
-    optServiceNameKey -> journeyConfig.pageConfig.optServiceName,
+    optServiceNameKey -> journeyConfig.pageConfig.optLabels.flatMap(_.optEnglishServiceName)
```

Comment:
Potentially here, if we're passing through the English Service Name inside the Labels key, then we don't need this here.

Reply:
true, good spot, I will update.